components/color/ColorSelector.js

```diff
@@ -4,18 +4,16 @@ import { Icon, Input } from 'react-components';
 import { LABEL_COLORS } from 'proton-shared/lib/constants';
 
 import './ColorSelector.scss';
+import { classnames } from '../../helpers/component';
 
 const ColorSelector = ({ selected, onChange, className, colors = LABEL_COLORS }) => {
-    const getClass = (className, more) => {
-        return [className, more].filter(Boolean).join(' ');
-    };
     return (
-        <ul className={getClass('ColorSelector-container unstyled', className)}>
+        <ul className={classnames('ColorSelector-container unstyled', className)}>
```

Comment:
the array brackets are missing here: `classnames(['ColorSelector-container unstyled', className])`